test/integ/agent/end-to-end.integ.ts

```diff
@@ -57,7 +58,7 @@ test(
 
     const doWork = metricScope(metrics => () => {
       metrics.putDimensions(dimensions);
-      metrics.putMetric(metricName, 100, 'Milliseconds');
+      metrics.putMetric(metricName, 100, 'Milliseconds', StorageResolution.High);
```

Comment:
Can we make this one `Standard` since the one on line 37 is `High`.

Reply:
Done.